ms-utils/properties/CombustFactory.cpp

```diff
@@ -44,11 +47,15 @@ Combust* CombustFactory::createCombustor(CombustFactory::CombustorType Type){
     case CombustFactory::NONE :
         return 0 ;
         break ;
-    case CombustFactory::CH4 :
+    case CombustFactory::CH4 : {
         CombustCH4* combustor = new CombustCH4() ;
         mCreatedCombustors.push_back(combustor) ;
         return combustor ;
         break ;
     }
+    default:
+        TS_HS_EXCEPTION(TS_HS_FATAL, TS_HS_GENERIC, " Combustor Type is an invalid value.",
```

Comment:
Change to TS_HS_ERROR, and TsOutOfBoundsException.

Reply:
Done.